tests/CMakeLists.txt

```diff
@@ -125,7 +126,7 @@ ROBOPTIM_TRAJECTORY_TEST(vector-interpolation)
 IF(TESTSUITE_SOLVER)
   # TODO: fix these tests and re-add them
   #ROBOPTIM_TRAJECTORY_TEST(spline-bicycle-optim)
-  #ROBOPTIM_TRAJECTORY_TEST(spline-optimization)
+  ROBOPTIM_TRAJECTORY_TEST(spline-optimization)
```

Comment:
Any particular reason for re-adding this in this PR? According to Travis this will require [some work](https://travis-ci.org/roboptim/roboptim-trajectory/jobs/61402581#L8265) unrelated to spline plotting.